packages/manager/modules/pci/src/projects/project/storages/cold-archive/containers/flush-archive/flush-archive.controller.js

```diff
@@ -0,0 +1,77 @@
+import {
+  COLD_ARCHIVE_DEFAULT_REGION,
+  TERMINATE_INPUT_PATTERN,
+} from './flush-archive.constants';
+
+import { COLD_ARCHIVE_CONTAINER_STATUS } from '../containers.constants';
+
+export default class ColdArchiveContainersFlushArchiveController {
+  /* @ngInject */
+  constructor($translate, PciStoragesColdArchiveService) {
+    this.$translate = $translate;
+    this.pciStoragesColdArchiveService = PciStoragesColdArchiveService;
+  }
+
+  $onInit() {
+    this.trackPage('containers::container::flush-archive');
```

Comment:
**question:** I see the default tracking page is renamed in the state definition. Why the page is tracked again here?

Reply:
I removed the tracking call.